pairing.php

```diff
@@ -45,8 +47,28 @@ function calculate(PDO $pdo): self
     }
 
     //TODO: generate sql to add the pairing to completed rounds
-    function sql($id, $i1, $i2): string
+    function sql(PDO $pdo, string $id, item $i1, item $i2): string
     {
-        throw new Error("sql not implemented");
+        $insertSql = "INSERT INTO completed_rounds (tournament_id, item1_id, item2_id, player, winner) 
```

Comment:
Remember, there's no completed rounds table. theres an `<id>_h2h` table which stores the rounds.